### PR TITLE
Minor fixes and additions

### PR DESCRIPTION
**Pull request by nerdlet**
## Description
#167639943

Fixes # (issue)

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by kilemensi — COMMENTED** [review]
Any reason for the `/favicons/` to `/src/favicons/` change @nerdlet?

**Comment by nerdlet** [comment]
@kilemensi Made necessary changes in my git config settings to fix git dif showing multiple changes in one file. Also, could you check if it's working on your end? So that I can determine if we need to add ` .gitattributes `  file to deal with the inconsistencies as per this link
https://help.github.com/en/articles/configuring-git-to-handle-line-endings

**Comment by kilemensi** [comment]
What I mean @nerdlet is for example, see the diff below:
Screenshot from 2019-08-05 14-58-17

The two codes are exactly the same. It's just the one on the right, you've formatted it using new style compared to the one on the left. These formatting changes are the ones that I'm saying we should find a way to live without since they only make our code differ from feinstaub's _unnecessarily_. It's not line ending...

File: src/index.html [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
       sizes="180x180"
-      href="/favicons/apple-touch-icon.png"
+      href="favicons/apple-touch-icon.png"
     />
     <link
       rel="icon"
       type="image/png"
       sizes="32x32"
-      href="/favicons/favicon-32x32.png"
+      href="favicons/favicon-32x32.png"
```
Comment (kilemensi) [inline src/index.html]: All  favicons `href` should start with `/` i.e. `/favicons/...`

**Comment by nerdlet** [comment]
@kilemensi Done

**Review by kilemensi — APPROVED** [review]
:+1: @nerdlet

**Pull request merged**